spk/homeassistant/Makefile

```diff
@@ -11,16 +11,16 @@ BUILD_DEPENDS += cross/pyephem cross/netifaces cross/psutil
 
 SPK_DEPENDS = "python3>=3.4.1-4:git"
 
-DEPENDS = cross/busybox cross/nmap #for python-libnmap
+DEPENDS = cross/$(SPK_NAME) cross/busybox cross/nmap #for python-libnmap
 
 MAINTAINER = SynoCommunity
-DESCRIPTION = Home Assistant is an open-source home automation platform running on Python 3. Track and control all devices at home and automate control.
+DESCRIPTION = "Home Assistant is an open-source home automation platform running on Python 3. Track and control all devices at home and automate control. The default password is empty, just click \"Login\".  Edit your settings via ssh in the configuration.yaml file. This is a beta package. Hue and Chrome are tested. Other modules might work, please test them and try to contribute if something does not work."
 ADMIN_PORT = 8123
 DISPLAY_NAME = Home Assistant
 DSM_UI_DIR = app
 
 BETA = 1
-CHANGELOG = "First release"
+CHANGELOG = "Update to revision bfa8131f4bff33160e08df3a2e0ead76d53f2728. PyISY will now work without having the climate module installed. More release notes at https://home-assistant.io/blog/2015/07/11/ip-cameras-arduino-kodi-efergy-support/."
```

Comment:
You can use the shorthand notation for the revision, no need to repeat the whole sequence. `bfa8131` would do. 
With separate entries, use `1.`,`2.` to separate entries. Use `<br>` for the line ending. See other packages for examples.